zookeeper-server/src/main/java/org/apache/zookeeper/server/command/MonitorCommand.java

```diff
@@ -80,6 +81,20 @@ public void commandRun() {
             print("max_proposal_size", leader.getProposalStats().getMaxBufferSize());
             print("min_proposal_size", leader.getProposalStats().getMinBufferSize());
         }
+
+        ServerMetrics.getMetrics()
+                    .getMetricsProvider()
+                    .dump(
+                    (metric, value) -> {
+                        if (value instanceof Long
+                                || value instanceof Integer) {
+                            print(metric, ((Number) value).longValue());
+                        } else if (value instanceof Number) {
+                            print(metric, ((Number) value).doubleValue());                        
+                        } else {
+                            print(metric, value + "");
```

Comment:
Is the `+ ""` an alternative form of `.toString()`? If so, I'd prefer the latter.

Reply:
I did so in order to not have problems with null values.
I will add a null check and call toString.